src/Neves/Events/Concerns/DelegatesToDispatcher.php

```diff
@@ -7,11 +7,11 @@ trait DelegatesToDispatcher
     /**
      * Register an event listener with the dispatcher.
      *
-     * @param  string|array $events
-     * @param  mixed $listener
+     * @param  \Closure|string|array  $events
+     * @param  \Closure|string|null  $listener
      * @return void
      */
-    public function listen($events, $listener)
+    public function listen($events, $listener = null)
```

Comment:
I believe I'll need to tag it as a major version, since the API changed. It won't work for Laravel 7 if no listener is passed.

Reply:
🤔

Are you sure?

Maybe I got it backwards, but to me this means:
- Laravel < 8 (i.e. 5/6/7) => `$listener` has to be passed explicitly
- Laravel 8 => `$listener` is optional

Since it _became_ optional, it's backwards compatible, no?

Reply:
Dang. Got it, I think. What id you someone extends _this_ library… right?

Reply:
> Since it _became_ optional, it's backwards compatible, no?

It became optional, which means that Laravel 7 would get `null` in $listener. `null` are unexpected in the `makeListener` function (see https://github.com/laravel/framework/blob/7.x/src/Illuminate/Events/Dispatcher.php#L366). 🤔 

> Dang. Got it, I think. What id you someone extends this library… right?

Precisely, for people using this library, IDE will not not require explicit `$listener` in the listen method. That would make Laravel to explode 💥